### PR TITLE
S3 Browser refactor to not use VFS

### PR DESCRIPTION
**Pull request by hunterwerlla**
## Description
- Remove the VFS from S3 browser
- Add a "Load More..." to solve the pagination issue
- Clicking on the same bucket again will focus it instead of opening another one
- Fix tests

## Screenshots
Screen Shot 2019-12-10 at 4 10 03 PM

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kiiadi** [comment]
Feature Request: right-click action on a 'directory' that allows you to 'load all'

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/S3Fields.kt [hunk 1]
```diff
@@ -14,9 +15,23 @@ sealed class S3Item(val key: String) {
 
 class S3Object(key: String, val size: Long, val lastModified: Instant) : S3Item(key)
 
+class S3ContinuationToken(key: String, val continuationToken: String) : S3Item(key)
```
[the remark this answers is not in the document]
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/S3Fields.kt]: This class is actually now deleted, this was here before the refactor

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/S3Utils.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.s3
+
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+
+object S3Utils {
```
[the remark this answers is not in the document]
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/S3Utils.kt]: Moved into another existing class, this function is very tied to the UI and doesn't really make sense in `core`

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties [hunk 3]
```diff
@@ -176,6 +176,7 @@ s3.download.object.action=Download...
 s3.download.object.failed=Failed to download object
 s3.download.object.description=Download
 s3.download.object.progress=Downloading ''{0}''
+s3.load_more=Load More...
```
Comment (kiiadi) [inline resources/resources/software/aws/toolkits/resources/localized_messages.properties]: nit: should this be title case? feel like it should be `Load more...` or even `load more...`

Comment (hunterwerlla) [reply resources/resources/software/aws/toolkits/resources/localized_messages.properties]: Agreed, it probably should not be title case

**Pull request merged**